common/lib/xmodule/xmodule/templates/problem/jsinput_response.yaml

```diff
@@ -26,13 +50,7 @@ data: |
       ]]>
       </script>
       <p>
-      The shapes below can be selected (yellow) or unselected (cyan).
-      Clicking on them repeatedly will cycle through these two states.
-      </p>
-      <p>
-      If the cone is selected (and not the cube), a correct answer will be
-      generated after pressing "Check". Clicking on either "Check" or "Save"
-      will register the current state.
+      In the following image, click the cone until it turns yellow.
```

Comment:
FYI, I clicked the cone, it turned yellow, and then when I clicked Check it gave me the dreaded red X. huh?